backend/muster-roll/src/main/java/org/egov/service/NotificationService.java

```diff
@@ -38,17 +38,14 @@ public class NotificationService {
      */
     public void sendNotificationToCBO(MusterRollRequest musterRollRequest){
         String action = musterRollRequest.getWorkflow().getAction();
-        if(action.equalsIgnoreCase("SENDBACKTOCBO") || action.equalsIgnoreCase("RE-SUBMIT")
-            || action.equalsIgnoreCase("APPROVE")) {
+        if(action.equalsIgnoreCase("SENDBACKTOCBO") || action.equalsIgnoreCase("APPROVE")) {
```

Comment:
@shubhang-eGov keep all the action codes as constants. Will be easier to change later also.